cfme/utils/trackerbot.py

```diff
@@ -235,23 +240,36 @@ def post_jenkins_result(job_name, number, stream, date, template, build_status,
         print(exc.content)
 
 
-def trackerbot_add_provider_template(stream, provider, template_name, custom_data=None):
+def add_provider_template(stream, provider, template_name, custom_data=None, mark_kwargs={}):
```

Comment:
I'd avoid using mutual object here.  it shouldn't cause something bad here but sometimes it leads to unexpected results. especially when you modify it inside function.